pkg/api/vlabs/validate_test.go

```diff
@@ -284,7 +284,7 @@ func Test_OrchestratorProfile_Validate(t *testing.T) {
 
 	for testName, test := range tests {
 		t.Run(testName, func(t *testing.T) {
-			t.Parallel()
+			//t.Parallel()
```

Comment:
Is this required? I tested this test case change and it seems to work consistently with t.Parallel() enabled.